types/txn.go

```diff
@@ -80,6 +79,28 @@ func NewTxParseContext(chainID uint256.Int) *TxParseContext {
 	return ctx
 }
 
+// recover sender address after appropriately populating ctx.Sighash, ctx.R & ctx.S
+func (ctx *TxParseContext) RecoverSender(vByte byte, sender []byte) error {
+	binary.BigEndian.PutUint64(ctx.Sig[0:8], ctx.R[3])
```

Comment:
curious about the general encoding choices, I saw both LittleEndian & BigEndian used, is that right?

Reply:
Unfortunately yes.. ssz uses the opposite of RLP/legacy encoding.